app/ts/components/simulationExplaining/customExplainers/GovernanceVoteVisualizer.tsx

```diff
@@ -57,7 +75,8 @@ export function GovernanceVoteVisualizer(param: TransactionImportanceBlockParams
 					{ simulateGovernanceContractExecutionReply !== undefined ? <>
 						<button
 							class = { `button is-primary is-small` }
-							onClick = { simulateGovernanceVote }>
+							onClick = { simulateGovernanceVote }
+						>
 							Refresh
 						</button>
 					</> : <></>}
```

Comment:
Consider creating locally scoped components to improve readability:
Earlier in this function (after you set `simulateGovernanceVote` and `simulateGovernanceContractExecutionReply`) add something like this:
```ts
const [MaybeRefreshButton_] = useState(() => () => simulateGovernanceContractExecutionReply !== undefined ? <button class = { `button is-primary is-small` } onClick = { simulateGovernanceVote }>Refresh</button> : <>)
```
Then here you would put:
```suggestion
					<MaybeRefreshButton_/>
```
The nice thing about function-local components is that you can reference captured variables so you don't need to have properties on the component and pass a bunch of stuff around.

Note: The `[]` and the doubly-nested lambda are both necessary.  The `_` suffix on the variable name is a style I use in my personal projects to help add clarity when a component is function-local vs file-local or imported, as well as to avoid naming collisions since for these local functions I often find myself wanting to name them things like `Button` or `Refresh` which often collide with existing names of things.

It may be worth multi-lining the button definition above if the one-liner is too noisy in this case.